doc/features/pre-existing-tenant.md

```diff
@@ -0,0 +1,59 @@
+# Pre-existing Tenant Feature
+
+# Table of Contents
+
+- [1. Overview](#1-overview)
+- [2. Motivation](#2-motivation)
+- [3. Mechanism](#3-mechanism)
+- [4. Lifecycle](#4-lifecycle)
+- [5. Example](#5-example)
+
+## 1. Overview
+
+This feature is introduced to enable adding a Kubernetes cluster to a pre-existing tenant like the common tenant on APIC. The APIC objects provisioned for different clusters should be uniquely named and distinguishable. When deleting a cluster, the objects created by other clusters shouldn’t be affected.
```

Comment:
- like for example the _common_ or any other pre-existing tenants
- Why we say _shouldn’t be affected_ we should say won't be affected.